src/AcceptanceTests/Routing/When_scaling_out_senders_that_uses_callbacks.cs

```diff
@@ -71,8 +71,8 @@ public class ScaledOutClient : EndpointConfigurationBuilder
         {
             public ScaledOutClient()
             {
-                EndpointSetup<DefaultServer>(endpointConfiguration =>
-                    endpointConfiguration.ConfigureTransport().Routing().RouteToEndpoint(typeof(MyRequest), typeof(ServerThatRespondsToCallbacks)));
+                EndpointSetup<DefaultServer>(c =>
```

Comment:
Personally, I see this as unnecessary change. The name was descriptive, and now it's "go figure". "c" as in "context"? Please revert.